### PR TITLE
show proxy "Order for" text if flag set

### PR DESCRIPTION
**Pull request by KeithLyonMastek**
Story [AB#12958](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/12958)
Task [AB#13821](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13821)

### CODE REVIEW

File: app/config.js
Comment (bjss-chris) [inline app/config.js:77]: I think it might be best to leave this to default to true, unless you think it could cause any problems?

**Review by bjss-chris — COMMENTED** [review]
Looks good to me – just one suggestion/question on the change of the default flag value.

File: app/config.js
Comment (KeithLyonMastek) [reply app/config.js:77]: The flag should only become true if the logged in user is a proxy buyer.  By default, they wont be

Comment (bjss-chris) [reply app/config.js:77]: Not sure I follow – the user's credentials won't have any effect on the flag. However, the story doesn't make it clear what the expected behaviour is for a non-proxy user, e.g. whether the dashboard should appear exactly as it does now, which would make sense but best not to assume – currently waiting for a response from Emily.

If the above is true then we'll need to add an extra condition to the template to make sure the proxy-version of the dashboard is only rendered for proxy users. Once the API changes have been completed the user object will have a `secondaryOrganisation` claim if they are a proxy buyer (`req.user.secondaryOrganisation`).

In dev, I think it will be best if we keep the default value of the flag as true though.

Comment (KeithLyonMastek) [reply app/config.js:77]: The ticket's requirements have been met and by having it as false ensures no scope creep

Comment (KeithLyonMastek) [reply app/config.js:77]: so the logic should be that is user credentials says your a proxy, show the proxy information.  

And the flag should just be "if flag is true, and if usercredentials.isProxy" then show?

Comment (bjss-chris) [reply app/config.js:77]: The flag should just be true or false (set by environment variable) or a default value if no environment variable present, which can be true as we'll explicitly set an environment variable in the non-dev environments.

Therefore, the condition that determines whether the proxy information is shown should be 'flag is true and user is proxy', e.g. something akin to `showProxy && req.user.secondaryOrganisation`.

Comment (KeithLyonMastek) [reply app/config.js]: I have implemented what I mentioned above, the if flag is true, and if usercredentials.isProxy", by creating a dummy variable (as the API has not been finished yet with the actual variable) to hold the 'usercredentials.isProxy' value

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**
